ansible/roles/dropbear_initramfs/tasks/main.yml

```diff
@@ -23,7 +23,7 @@
 - name: Configure dropbear options
   ansible.builtin.template:
     src: 'etc/dropbear-initramfs/config.j2'
```

Comment:
Move the source files also to match new version of dropbear.

Reply:
You may base your work on https://github.com/ypid/debops/tree/feat/dropbear-debian-bookworm.

Reply:
- [x] Moved `etc/dropbear-initramfs/config.j2` (old template) -> `etc/dropbear/initramfs/dropbear.conf.j2`
- [x] Updated `src` path within `ansible/roles/dropbear_initramfs/tasks/main.yml`